NBISweden/urdr
# Migrate from snowpack to webpack

## Reason for this PR
The snowpack project is not under active maintenance any longer so it's a good idea to switch to a widely adopted tool such as webpack.



## Type of change

 
## List of changes made

- Producttion build works with webpack
- Dev server with hot reload powered by webpack

## Testing

1) Bring down all containers and volumes
2) Update your env accordingly
3) Build prod or dev setup
4) Start the services and navigate to localhost:4567
5) Test updating the css code and see if HR works

## Definition of Done checklist
